.github/sigstore-python-conformance

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+"""
+A wrapper to convert `sigstore-conformance` CLI protocol invocations to match `sigstore-python`.
```

Comment:
Separate tiny nit: could we expand this comment to include a link to the `sigstore-conformance` repo?

Reply:
[3da269a](https://github.com/sigstore/sigstore-python/pull/298/commits/3da269ad139b52ead77b627f564cea68e0b9abd7)